sql/core/src/test/resources/sql-tests/results/inline-table.sql.out

```diff
@@ -166,18 +166,18 @@ struct<plan:string>
 
 == Analyzed Logical Plan ==
 col1: string, col2: int, col1: string, col2: int
-Project [col1#x, col2#x, col1#x, col2#x]
+Project [col1#x: string, col2#x: int, col1#x: string, col2#x: int]
 +- Join Cross
-   :- LocalRelation [col1#x, col2#x]
-   +- LocalRelation [col1#x, col2#x]
+   :- LocalRelation [col1#x: string, col2#x: int]
+   +- LocalRelation [col1#x: string, col2#x: int]
```

Comment:
Repeatedly showing the data types for the same attributes may not useful. Seems too verbose. For a big query plan, it will be filled with redundant info.